artifactory/services/movecopy.go

```diff
@@ -109,7 +108,9 @@ func (mc *MoveCopyService) moveFiles(regexpPath string, resultItems []utils.Resu
 				destPathLocal = clientutils.TrimPath(destPathLocal + "/" + v.Path + "/")
 			}
 		}
-		destFile, err := clientutils.ReformatRegexp(regexpPath, v.GetItemRelativePath(), destPathLocal)
+		givenPath := clientutils.CleanRepoFromPath(params.GetFile().Pattern)
+		actualPath := clientutils.CleanRepoFromPath(v.GetItemRelativePath())
+		destFile, err := clientutils.ReformatTargetByPaths(givenPath, actualPath, destPathLocal)
```

Comment:
The above 3 lines seem to be duplicated (see the *download.go*).
I suggest moving them into *utils.go* (after consolidating them into one function).